core/src/main/java/org/apache/accumulo/core/cli/ClientOpts.java

```diff
@@ -92,6 +94,20 @@ public Password(String dfault) {
     public String toString() {
       return new String(value, UTF_8);
     }
+
+    /**
+     * Prompts user for a password
+     *
+     * @return user entered Password object, null if no console exists
+     */
+    public static Password promptUser() throws IOException {
+      if (System.console() == null) {
+        return null;
```

Comment:
I'm thinking that throwing an exception here would be more clear. This method cannot properly function as the caller expected, so they should get a clear exception in this case (not just a null return value).